src/popup/router/components/AssetSelector.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <select
+    class="asset-selector"
+    :value="JSON.stringify(currentToken)"
```

Comment:
This is not my code. You see this change because the AssetSelector wasn't merged when this PR was created. If you think this is a bug please create separate task for this.